README.md

```diff
@@ -0,0 +1,27 @@
+# CAOTIC
+High Contrast and Adaptive Optics laboratory testbeds community website
+
+Welcome to the community website for astronomical testbeds! We are currently working on getting our 
+website migrated to this place here, the original website can be found here:  
+https://sites.google.com/view/highcontrastlabs/
+
+## Local setup - MacOS
+
+### Initial setup
+
+1. install homebrew  
+https://osxdaily.com/2018/03/07/how-install-homebrew-mac-os/
+
+2. install ruby  
+`$ brew install ruby`
+
+3. install bundler  
+`$ gem install bundler`
+
+4. install Jekyll  
+`$ gem install bundler jekyll`
+
+### Render the website locally
+
+1. check out the branch that you want to build locally
+2. `$ jekyll serve`
```

Comment:
Do I need to be in a specific directory when I run this?

Reply:
yes, in the repo 😬

Reply:
Does it matter what level in the repo you need to be in? (I am waiting for things to install so I am just thinking ahead)

Reply:
No, that shouldn't matter.

Reply:
help!

<img width="807" alt="Screen Shot 2021-03-25 at 1 51 02 PM" src="https://user-images.githubusercontent.com/17128221/112519870-2c5c4f00-8d71-11eb-867c-16dd1a2c8bd0.png">

Reply:
How did point 4 from the section before go? Did jekyll install correctly?

Reply:
Yes it did. (no errors)

Reply:
Ok... let me pull up my local notes.